src/components/Result/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Result = () => (
+const Result = ({ operate }) => (
```

Comment:
What exactly does the bool `operate`?

Reply:
When you clic en Result button, this change to TRUE and I have to operate all operation that the user inserted